conda_package_streaming/exceptions.py

```diff
@@ -0,0 +1,21 @@
+class InvalidArchiveError(Exception):
+    """Raised on OSError"""
+
+    def __init__(self, msg, *args, **kw):
+        msg = (
+            "Error with archive. You probably need to delete and re-download "
+            "or re-create this file. Message was:\n\n%s" % (msg,)
+        )
+        super(InvalidArchiveError, self).__init__(msg)
```

Comment:
```suggestion
        super().__init__(msg)
```

Reply:
pre-commit seems more "aggressive" on futurize than I remember. Not a fan. But it does automatically make this change.